lib/options.py

```diff
@@ -15,6 +15,14 @@ def env_int(name, default):
         return default
 
 
+def env_string(name, default):
+    try:
+        value = os.environ.get(name)
+        return default if value is None else value
+    except ValueError:
+        return default
```

Comment:
`ValueError` is not possible here. Can be rewritten this way:

```python
return os.environ.get(name, default)
```

However we don't need to introduce the special function for this. It is enough to just pass `default=os.environ.get("TEST_RUN_MEMTX_ALLOCATOR", "memtx")` to `add_arguments()`.

Reply:
Fixed.